README.md

```diff
@@ -233,10 +233,10 @@ twig:
             - SimpleThingsFormExtraBundle:Form:field_type_help.html.twig
 ```
 
-You can also load the help extension form theme into your own form theme.
+Next, load the help extension form theme into your template.
```

Comment:
@stof can you please help me figure out what went wrong?

I followed the instructions like they were, which lead me to absolutely nothing.
It's only after changing
`{% use 'SimpleThingsFormExtraBundle:Form:field_type_help.html.twig' %}`
to
`{% form_theme form 'SimpleThingsFormExtraBundle:Form:field_type_help.html.twig' %}`
that the help messages showed up.

This made me believe it was the proper way to do. What could I have done wrong or what could i have missed?

Greetings,

Barthy